python/paddle/fluid/tests/unittests/test_conv2d_op.py

```diff
@@ -355,7 +355,7 @@ def test_check_output(self):
             place, atol=1e-5, check_dygraph=(self.use_mkldnn == False))
 
     def test_check_grad(self):
-        if self.dtype == np.float16:
+        if self.dtype == np.float16 or hasattr(self, "no_need_check_grad"):
```

Comment:
please also make sure the value of the `no_need_check_grad` attribute ist `True`
here, and below

Reply:
Done.